ts-packages/web/src/app/teams/[username]/_components/team-sidemenu.tsx

```
@@ -71,15 +71,15 @@ export default function TeamSidemenu({ username }: TeamSidemenuProps) {
+            <div className="w-20 h-20 rounded-full bg-profile-bg" />
           )}
         </div>
 
-        <div className="font-medium">{user.nickname}</div>
+        <div className="font-medium text-text-primary">{user.nickname}</div>
 
         <div
           id="user-profile-description"
-          className="text-xs text-gray-400"
+          className="text-xs text-desc-text"
           dangerouslySetInnerHTML={{ __html: user.html_contents }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)